runtime/dart_snapshot.cc

```diff
@@ -68,10 +68,17 @@ static std::shared_ptr<const fml::Mapping> SearchMapping(
     static const uint8_t* vm_isolate_instrs = nullptr;
     if (leaked_elf == nullptr) {
       const char* error = nullptr;
-      leaked_elf = Dart_LoadELF(native_library_path.back().c_str(), 0, &error,
-                                &vm_snapshot_data, &vm_snapshot_instrs,
-                                &vm_isolate_data, &vm_isolate_instrs,
-                                /* load as read-only, not rx */ true);
+      // vmcode files are elf files prefixed with a shorebird linker header.
+      auto elf_mapping =
+          GetFileMapping(native_library_path.back(), false /* executable */);
+      int elf_file_offset = Shorebird_ReadLinkHeader(elf_mapping->GetMapping(),
+                                                     elf_mapping->GetSize());
+
+      leaked_elf =
```

Comment:
Reviewing this code for the first time, it wasn't clear that "leaked" here meant that it was being leaked in the memory sense. Maybe an explanatory comment?